airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -184,15 +184,19 @@ def get_conn(self) -> BlobServiceClient:
             # connection_string auth takes priority
             return BlobServiceClient.from_connection_string(connection_string, **extra)
 
+        account_url = (
```

Comment:
That ended up still looking pretty gnarly, didn't it?  :disappointed:

Reply:
Yeah but quite understandable